build.gradle.kts

```diff
@@ -197,6 +197,8 @@ val prodAudience = mapOf(
     "figure_lending" to p8eParty("0A410417D207BF9381E4288CFA970E9708C13EE047882113D342E2F62C74D391811AF7059A36687BC2A2F3A33DBD0A689F0C8F937FB8301EE5416BC025C512D484A44D"),
     "figure_servicing" to p8eParty("0A410480031287B7BCBAEF26A81A408D167F948A386FCB1EE56B3BD477D5AE4179D9D7B59C6E7E791A0BB94A11BFF7EA0A5D2CAECFA8F2DDC566C83012AEE256670FF9"),
     "figure_servicing_old" to p8eParty("0A410457C9A540DCC947F2715C4D22437C127EC0A67A3A4F77533F5D0A94BAF6BE93CC7D870EAECBC212601428A7194D4F0182C89AF0665899F11004D839205858B9B9"),
+    "homebridge" to p8eParty("0A4104F6A08558B4F67A684176C505608724417505EA5B8CD61F32AA408C66B20B1F5E1EE1995DEAEBF05525733E207A7C3F13B39ECAADF1D7DF4D7128B000E98F1FFB"),
+    "guaranteed_rate" to p8eParty("0A41043661353B512C4A8303D40E926E03319D6054B26940AB1FDC8EF357EB5855DB02E62244B759D73D6D7781B78B055F2EDD673094FA79A007785E290D47DD87F8FF"),
```

Comment:
I'm pretty sure the keys don't matter in these maps, but do correct me if I'm wrong